clients/pkg/promtail/client/client.go

```diff
@@ -162,6 +162,9 @@ type Tripperware func(http.RoundTripper) http.RoundTripper
 
 // New makes a new Client.
 func New(metrics *Metrics, cfg Config, streamLagLabels []string, logger log.Logger) (Client, error) {
+	if cfg.StreamLagLabels.String() != "" {
+		level.Warn(logger).Log("msg", "client config stream_lag_labels is deprecated in favour of the config file options block field, and will be ignored", "stream_lag_labels", cfg.StreamLagLabels.String())
```

Comment:
```suggestion
		level.Warn(logger).Log("msg", "client config `stream_lag_labels` is deprecated in favour of the config file options block field, and will be ignored", "stream_lag_labels", cfg.StreamLagLabels.String())
```

small nit

Reply:
Hhmm, it's a pity the `options` setting didn't make it into v2.5.0. We should really link to the docs here; I don't think a user with minimal context would understand this warning message.

Reply:
@kavirajk do you really think we should be using backtick formatting here? log output in someones terminal won't necessarily display that in a way that would provide any extra value to the user.

Reply:
@cstyan good point. feel free to ignore 👍